decidim/decidim
# Do not display an error when filling a form

#### :tophat: What? Why?
This pull request addresses the issue of form validation errors appearing prematurely in the login form. The primary goal is to prevent error messages from showing up when a single character is entered into the input field until the field loses focus.

#### :pushpin: Related Issues
Fixes #12364

#### Testing

- Go to the login page.
- Click on the email input field.
- Enter a single character in the email input field.
- Ensure that no error message is displayed until the input field loses focus (i.e., you switch to another field).

:hearts: Thank you!
